src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1148,6 +1152,27 @@ public void cannotCheckOutWhenItemLimitIsReachedForBookMaterialTypeAndLoanTypeIs
     assertThat(secondBookTypeItem, hasItemStatus(AVAILABLE));
   }
 
+  @Test
+  public void canCheckOutWithdrawnItem() {
+    final IndividualResource withdrawnItem = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::withdrawn);
+
+    assertThat(withdrawnItem.getJson(), withdrawn());
+
+    final IndividualResource response = loansFixture
+      .checkOutByBarcode(withdrawnItem, usersFixture.steve());
+
+    assertThat(response.getJson(), allOf(
+      hasOpenStatus(),
+      hasItem(),
```

Comment:
What about the item is especially interesting?

Reply:
It is not so important for our case, so I've removed this assertion.